src/applications/personalization/profile360/util/helpers.jsx

```diff
@@ -31,6 +31,23 @@ export const formTitles = Object.keys(formBenefits).reduce((titles, key) => {
   return titles;
 }, {});
 
+export const formDescriptions = Object.keys(formBenefits).reduce(
+  (titles, key) => {
+    let formNumber;
+    if (key === '40-10007' || key === 'VIC') {
+      formNumber = '';
+    } else if (key === '1010ez') {
+      formNumber = ' (10-10EZ)';
```

Comment:
does this need a space?

Reply:
Good idea, @rianfowler, we could move the space to be after application, that seems clearer.